pr-remotes.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import sys
+import archive_reference
+
+parser = argparse.ArgumentParser(description="Identify the source repos and forks of PRs from an archive")
+parser.add_argument("input", help="GitHub archive file")
+parser.add_argument("git_dir", help="Local directory with git repo", default=".")
+parser.add_argument("--include-merged", dest="merged",
+    default=False,
+    action="store_true",
+    help="include merged PRs in analysis",
+)
+parser.add_argument("--dry-run", dest="dryRun",
+    default=False,
+    action="store_true",
+    help="Show what would be done, without making any changes"
+)
+parser.add_argument("repo", help="target GitHub repo")
+args=parser.parse_args()
+
+reference = archive_reference.loadReference(args.input, args.repo)
+
+target_prs = reference.prs.values() if args.merged else (pr for pr in reference.prs.values() if pr["state"] != "MERGED")
+commits = {pr["headRefOid"] + ":pr" + str(pr["number"]) for pr in target_prs}
```

Comment:
Other places I've seen this use "pulls/33".  This keeps the pull requests in a single directory under `.git/refs/remotes/origin/pulls/`.

Reply:
Good suggestion.  Done.